packages/flame/lib/src/game/game.dart

```diff
@@ -216,6 +216,13 @@ abstract class Game {
   /// - GameWidget
   /// - [Game.overlays]
   final overlays = ActiveOverlaysNotifier();
+
+  /// Use this method in case you need to project a coordinate received by a
+  /// gesture/pointer event to your game coordinate system.
+  ///
+  /// By default this just returns the same received coordinate transformed to
+  /// a Vector2, override this to add addtional logic to that projection
+  Vector2 projectCoordinates(Offset value) => value.toVector2();
```

Comment:
I think we should do 1 of 2 things:
either
1. make this take a vector2
or
2. change the docs slightly to say: this is a method used internally by the engine that...

we shouldn't endorse people using non vector2 stuff

Reply:
Can you give me a suggestion on what to add on the docs?

Reply:
Maybe rename it to `projectOffset` so that it is even clearer, since all other coordinates should be Vector2?